File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = new int[2];
        int cnt=0;
        int win=0;
        for(int num:lottos){
            if(num==0){
                cnt++;
                continue;
            }
            for(int win_num:win_nums){
                if(num==win_num){
                    win++;
                    continue;
                }
            }
        }
        answer[0]=7-cnt-win;
        answer[1]=7-win;
        for(int i=0;i<2;i++){
            if(answer[i]==7)
                answer[i]--;
        }
        return answer;
    }
}

File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    
    z = lottos.count(0)
    c = 0
    
    for n in lottos:
        if n in win_nums:
            c += 1
    answer = []
    if z+c <=1:
        return [6,6]
    else:
        answer.append(7-z-c)
    
    if c <=1:
        return [answer[0],6]
    else:
        return [answer[0],7-c]